### PR TITLE
43 label a sentence

### PR DESCRIPTION
**Pull request by alimuhammadahmer**
This requests contains DSL file and a SQL file for labelling a particular sentence.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,3 +13,17 @@ https://github.com/buerokratt/Data-Anonymizer/blob/main/DSL.Ruuter/POST/regex.ym
...
 ```
+
+### Label a sentence
+
+https://github.com/buerokratt/Data-Anonymizer/issues/43
+
+https://github.com/buerokratt/Data-Anonymizer/blob/main/DSL.Resql/upsert_corpora_task.sql
+
+```
+```
```
Comment (turnerrainer) [inline README.md:24]: Please add a sample curl request.

File: README.md [hunk 2]
```diff
@@ -13,3 +13,17 @@ https://github.com/buerokratt/Data-Anonymizer/blob/main/DSL.Ruuter/POST/regex.ym
...
+
+https://github.com/buerokratt/Data-Anonymizer/blob/main/DSL.Ruuter/POST/annotate.yml
+
+```
+```
```
Comment (turnerrainer) [inline README.md:29]: Please add a sample curl request.

**Review by turnerrainer — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (alimuhammadahmer) [reply README.md]: Done

File: DSL.Resql/upsert_corpora_task.sql [hunk 3]
```diff
@@ -0,0 +1 @@
+insert into Corpora_Tasks (id, raw_text, corpora_id, is_private, preannotated_text, annotated_text, sentences_annotations) select :id, raw_text, corpora_id, is_private, preannotated_text, :annotated_text, :sentences_annotations::json from Corpora_Tasks where corpora_id=:corpora_id and id=:id ORDER BY created_at DESC limit 1 RETURNING id;
```
Comment (turnerrainer) [inline DSL.Resql/upsert_corpora_task.sql:1]: No newline at end of file

File: DSL.Ruuter/POST/annotate.yml [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+    url: http://resql:8082/upsert_corpora_task
+    body:
+      id: ${incoming.body.id}
+      annotated_text: ${incoming.body.annotated_text}
+      corpora_id: ${incoming.params.project}
+      sentences_annotations: ${incoming.body.sentences_annotations}
+  result: request_response
+
+return_value:
+  return: ${request_response.response.body}
```
Comment (turnerrainer) [inline DSL.Ruuter/POST/annotate.yml:13]: No newline at end of file

**Review by turnerrainer — CHANGES_REQUESTED** [review]
No newline at end of file, otherwise OK

**Review by turnerrainer — APPROVED** [review]

**Pull request merged**